src/block/block_read.c

```diff
@@ -173,27 +173,14 @@ __wti_block_read_off(WT_SESSION_IMPL *session, WT_BLOCK *block, WT_ITEM *buf, ui
     chunkcache_hit = full_checksum_mismatch = false;
     check_size = 0;
     failures = 0;
+    bufsize = WT_MAX(size, buf->memsize + 10);
```

Comment:
From my reading of the comment, the `+ 10` was done to ensure that a buffer without `WT_ITEM_ALIGNED` would be re-allocated after settting the `WT_ITEM_ALIGNED` flag. So is this `+ 10` still required?

Reply:
I wasn't certain when I made this change. The way I read it was, if the WT_ITEM_ALIGNED was set, then the size was already aligned (big enough for use), but if not then increase the size and set the flag (to indicate it is now big enough / aligned). I took the conservative route and since the flag is no longer in use (and we are no longer aligning with this flag), then I would just use the increased the size so that it is big enough.

Reply:
Thanks for the explanation! [b4451da](https://github.com/wiredtiger/wiredtiger/pull/10836/commits/b4451da9de447d775a9d2752c807af7ff20f95af)

Reply:
Most welcome.